.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,7 @@
 **Before creating a pull request make sure that:**
 
-- [ ] commit messages are meaningful and follow good commit message guidelines
+- [ ] PR's title includes JIRA ticket number
+- [ ] commit messages are meaningful and follow [Conventional Commits](https://www.conventionalcommits.org/) specification
```

Comment:
I think it is more the PR titles that follow that standard, I don't think we really care about the individual commit messages

Reply:
:+1: updated